src/collections/componentSet.ts

```diff
@@ -453,14 +453,15 @@ export class ComponentSet extends LazyCollection<MetadataComponent> {
    * @param forDestructiveChanges Whether to build a manifest for destructive changes.
    */
   public async getPackageXml(indentation = 4, destructiveType?: DestructiveChangesType): Promise<string> {
-    const j2x = new j2xParser({
+    const builder = new XMLBuilder({
       format: true,
       indentBy: new Array(indentation + 1).join(' '),
```

Comment:
clever, but the +1 feels awkward.

''.padEnd(indentation, ' ')
```suggestion
      indentBy: ''.padEnd(indentation, ' '),

```